### DISCUSSION TITLE
The horde, where can I check my current kudos?

### DISCUSSION THREAD

**Discussion by shawhu** [KoboldAI/KoboldAI-Client#331] [General]
I've managed to launched a woker and I saw tasks has been pulled and the generation has been sent. But...how do I check my kudos?

**Comment by henk717** [comment]
https://lite.koboldai.net can show it to you when you enter the API key, you can also join our discord at https://koboldai.org/discord and use /login to login to the horde statistics after which /userinfo also works.
